docs/manual/webhooks/advanced-run-job.md

```diff
@@ -69,48 +68,56 @@ like [Node Filter](#node-filter)
 Job options to be supplied during job execution. Use [JsonPath](#jsonpath) or
 [Templates](#templates) to construct the `value` from event data.
 
-### [Conditions](#conditions)
-See [Conditions](#conditions)
-
 ## Conditions
+
+Each rule can have multiple Conditions. Based on the [Policy](#policy), the rule will apply when all or any of the conditions are satisfied.  Each Condition defines a Field selector using JsonPath, a particular match type (defined below), and a value.
+
 ![Oh noes :O](../../assets/img/wh-routing-rule-conditions-overview.png)
 
 ### Policy
+
+When more than one Condition is added to the Rule, you can select which policy to use for evaluating the Conditions.
+
 * **all** All conditions must match
 * **any** At least one condition must match
 
-### contains
-Satisfied if the value at `path` contains the provided `value`.
-### dateTimeAfter
-### dateTimeBefore
-:::warning
-Must be a Date with Time.
-:::
-Compares the parsed Zoned DateTime at `path` with the provided DateTime.
-Supported formats are:
-* [ISO_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_DATE_TIME)
-* [ISO_INSTANT](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT)
-* Unix EPOCH (assumed to be in milliseconds if year >=2970)
-
-The format in the event need not match the format of the value.
-
-### exists
-Satisfied if _Something_ exists at the specified path.
-### matches
-Matches is satisfied if the value at `path` _equals_ the value of
+### Match Types
+
+`contains`
+:   Satisfied to true if the Field is equal to, or contains a substring equal to, the provided `value`.
+
+`dateTimeAfter`/`dateTimeBefore`
+:   Satisfied if the Field, parsed as a Zoned DateTime, compares with the provided DateTime.
+
+	:::warning
+	Must be a Date with Time.
+	:::
+
+	Supported formats for the Field and the value are:
+
+	* [ISO_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_DATE_TIME)
+	* [ISO_INSTANT](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT)
+	* Unix EPOCH (assumed to be in milliseconds if year >=2970)
+
+`exists`
```

Comment:
The thought process behind these being H3s was to get them into the sidebar:

![image](https://user-images.githubusercontent.com/271965/70954273-1d490600-2032-11ea-93bb-6f4b3d551106.png)

I'm not opposed to the change though if you are not.

Reply:
hm, yeah i guess it's just a style choice; i think it's not quite necessary, the entire list can be viewed in one page